src/app/components/splash-screen/splash-screen.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit, OnDestroy , ViewEncapsulation } from '@angular/core';
+import { UtilityService } from 'src/app/services/utility.service';
+import {DEFAULT_IMAGE_TEXT} from "src/constants/constant";
+
+@Component({
+  selector: 'app-splash-screen',
+  templateUrl: './splash-screen.component.html',
+  styleUrls: ['./splash-screen.component.scss'],
+  encapsulation: ViewEncapsulation.None
+})
+export class SplashScreenComponent implements OnInit , OnDestroy {
+  showSplash: boolean;
+  imgText= DEFAULT_IMAGE_TEXT;
+  timeOutID: number;
+  iconImageUrl = "../../../assets/icons/logo.svg";
+
+  constructor(private _utilityService: UtilityService) {}
+
+  ngOnInit() {
+    this.showSplash = this._utilityService.showSplash;
+    this.timeOutID =   window.setTimeout(() => {
```

Comment:
reference to the `window` object is probably redundant.

```suggestion
    this.timeOutID =  setTimeout(() => {
```

Reply:
vaia, I was trying to catch the return value in _timeoutId_  declared as of type number. And  using window.setTimeOut was returning the proper type whereas only setTimeOut was returning something that is not capturable to a number type variable. 

Following your sugggestion, I have declared _timeoutId: ReturnType<typeof setTimeout>_ this way. And removed window object. It's working now.